recipe/meta.yaml

```diff
@@ -62,3 +65,6 @@ extra:
     - SylvainCorlay
     - bollwyvl
     - jtpio
+
+skip-lints:
```

Comment:
For this skip to be effective it should be nested inside `about`.
I would prefer to remove this skip as it is only there because of a linter shortcoming.

Reply:
Thanks. I've just removed it